public/manifest.json

```diff
@@ -1,29 +1,14 @@
 {
-  "name": "WebMark",
+  "name": "__MSG_extensionName__",
   "version": "0.1.0",
-  "description": "Load a random page from your bookmarks!",
+  "description": "__MSG_extensionDescription__",
+  "default_locale": "en",
   "permissions": [
     "bookmarks",
     "notifications",
     "storage",
     "tabs"
   ],
-  "commands": {
-    "save": {
-      "suggested_key": {
-        "default": "Alt+S",
-        "mac": "Alt+S"
-      },
-      "description": "Save this page to bookmark"
-    },
-    "load": {
-      "suggested_key": {
-        "default": "Alt+Q",
-        "mac": "Alt+Q"
```

Comment:
Remind me to test these shortcuts out on a mac environment later

Reply:
https://github.com/jeongm-in/webmark/issues/13#issuecomment-508929700